examples/00_quick_start/sar_movieratings_with_azureml_designer.ipynb

```diff
@@ -1,14 +1,5 @@
 {
  "cells": [
```

Comment:
hey Blanca, some minor comments:

1) we need to leave the copyright
2) We are transitioning to change the principal branch from master to main, so `https://github.com/microsoft/recommenders/blob/master/reco_utils/azureml/azureml_designer_modules/module_specs/` should be `https://github.com/microsoft/recommenders/blob/master/reco_utils/azureml/azureml_designer_modules/module_specs/`
3) In this line there is an error with the SETUP link:
```
The first step is to install Azure CLI and Component CLI extension. Assuming that you have installed the Recommenders environment reco_base as explained in the [SETUP.md].
```
should be
```
The first step is to install Azure CLI and Component CLI extension. Assuming that you have installed the Recommenders environment reco_base as explained in the [SETUP.md](../../SETUP.md).
```